src/generate/generate_from_csv.py

```diff
@@ -128,8 +137,9 @@ def generate_from_csv(args):
                 next_arrival = v_id_list[idx + 1]["arrival_time"]
                 next_arrival = datetime.datetime.strptime(next_arrival, DATETIME_FORMAT)
             except IndexError:
+                # no departure event: stand until end of simulation
```

Comment:
```suggestion
                # no departure event: stand until end of simulation or until up to 8 hours after scenario if last arrival is very close to end of scenario
```

Reply:
suggested line too long, amended to `no departure: stand for 8h or until end of simulation (whichever comes later)`